napari_plugin_manager/base_qt_plugin_dialog.py

```diff
@@ -600,6 +601,17 @@ def _on_enabled_checkbox(self, state: Qt.CheckState) -> None:
         """
         raise NotImplementedError
 
+    def _on_bundle(self) -> bool:
```

Comment:
This is only about warning PyPI for now and other users might not have a concept of what a bundle is. So let's make it explicit:

```suggestion
    def _warn_pypi_install(self) -> bool:
```

Reply:
The docstring would need to be updated, naturally.